ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -283,35 +283,69 @@ FAN_SS_DRSM8X:
   - DRSM8X
 
 FAN_EDPM_ZHM_ZTM_LPS:
-  id: "16288085548559433728"
   description: "Exhaust fan with sensors (temperature, humidity and low pressure)."
   is_canonical: true
   implements:
   - FAN
   - EDPM
   - ZHM
   - ZTM
-  uses:
-  - run_command
-  - run_status
+  opt_uses:
+  - exhaust_fan_run_command
+  - exhaust_fan_run_status
```

Comment:
why are the `run_command` and `run_status` points optional? should be required.

Reply:
implement SS.